### DISCUSSION TITLE
Welcome to AOTG-Public-Issue-Board Discussions!

### DISCUSSION THREAD

**Discussion by DrewBokman** [DrewBokman/AOTG-Public-Issue-Board#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DrewBokman** [comment]
troll

**Comment by Aibra1337** [comment]
Username: robloxiaman194
Suggestion: Allow melee attacks to work while on horseback.
Reason: So that we're able to attack enemies instead of our horse.

**Reply by DrewBokman** [reply]
Wrong place, please use https://github.com/DrewBokman/AOTG-Public-Issue-Board/issues/new/choose
